user/views.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import render
+from .serializers import UserSerializer
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+# Create your views here.
+    
+@api_view(['POST'])
+def create_user(request):
```

Comment:
How are you handling password saving, I would expect the password to be encrypted, as it is not safe to store passwords as normal strings?